src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final int randomNumber = new Random().nextInt(101);
+        Ball randomBall = new Ball(new ColorSupplier().getRandomColor(), randomNumber);
+        return randomBall;
```

Comment:
```suggestion
public class Lottery {
    private static final ColorSupplier colorSupplier = new ColorSupplier();
    private static final Random random = new Random();
    
    public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(101));
```
would be better to extract objects like this to class fields
don't create a variable if you return the value immediately after assigning to that variable